### PR TITLE
Add PSL catalog JSON file

### PR DESCRIPTION
**Pull request by Peter-Metz**
This PR creates a new file called `PSL_catalog.json` that contains the information needed to generate a webpage for B-Tax on the PSL website. [Here](https://github.com/open-source-economics/PSL-Core/tree/master/Tools/Catalog-Builder) is some more information on the Catalog Builder and how it uses the JSON file to create a the webpage.

@jdebacker I'd appreciate your review to make sure all of the information is correct and that we're not leaving anything out.

### CODE REVIEW

File: PSL_catalog.json [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        "data": null
+    },
+    "license": {
+        "start_header": null,
+        "end_header": null,
+        "source": "https://github.com/open-source-economics/B-Tax/blob/master/LICENSE.md",
+        "data": "<p>CC0 1.0 Universal (CC0 1.0) Public Domain Dedication</p>",
+        "type": "html"
+    },
+    "user_documentation": {
```
Comment (jdebacker) [inline PSL_catalog.json]: @Peter-Metz You might link to this document here: https://github.com/open-source-economics/B-Tax/blob/master/Guides/CCC_Guide.pdf

**Comment by andersonfrailey** [comment]
@Peter-Metz I think you might have to merge in the latest changes from the master branch to this PR to get all the tests passing.

**Comment by Peter-Metz** [comment]
Thanks @andersonfrailey. Tests now pass

**Comment by jdebacker** [comment]
@Peter-Metz This looks good - let me know when it's ready to merge.

**Comment by Peter-Metz** [comment]
Thanks for looking this over @jdebacker. I noticed a slight quirk in the way the catalog builder renders markdown text, so I will try to fix that on the catalog builder side of things (shouldn't involve changing anything in B-Tax). Still, I think it makes sense to hold off on merging until we resolve this issue.

**Comment by Peter-Metz** [comment]
@jdebacker We sorted the rendering issue out so we're good to merge from my perspective

**Comment by jdebacker** [comment]
Thanks @Peter-Metz, merging now.

**Pull request merged**